pkg/discovery/probe/stitching/stitching_manager.go

```diff
@@ -0,0 +1,201 @@
+package stitching
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"k8s.io/kubernetes/pkg/api"
+
+	"github.com/turbonomic/kubeturbo/test/flag"
+
+	"github.com/turbonomic/turbo-go-sdk/pkg/builder"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+	"github.com/turbonomic/turbo-go-sdk/pkg/supplychain"
+
+	"github.com/golang/glog"
+)
+
+const (
+	// The property used for node property and replacement entity metadata
+	proxyVMIP   string = "Proxy_VM_IP"
+	proxyVMUUID string = "Proxy_VM_UUID"
+
+	// The default namespace of entity property
+	defaultPropertyNamespace string = "DEFAULT"
+
+	Stitch    stitchingPropertyType = "stitch"
+	Reconcile stitchingPropertyType = "reconcile"
+)
+
+type stitchingPropertyType string
+
+type StitchingManager struct {
+	// key: node name; value: node IP address for stitching.
+	nodeStitchingUIDMap map[string]string
+
+	// key: node name; value: node IP address for stitching.
+	nodeStitchingIPMap map[string]string
+
+	//
+	useVMWare bool
```

Comment:
define some other type of data, stitchIDType for example, to indicate whether IP is used, or UUID is used. And set this stitchIDType according to command line input or configuration.

Because we expect that in other cases (not only vmware), we may also need to use UUID for stitching.